src/base/emu-i386/simx86/sigsegv.c

```diff
@@ -266,6 +266,13 @@ int e_vgaemu_fault(sigcontext_t *scp, unsigned page_fault)
 
 /**/  e_printf("eVGAEmuFault: trying %08x, a=%08"PRI_RG"\n",*((int *)_rip),_rdi);
 
+    /* try CPatch, and if that fails, the exceptionally expensive route */
+#if 0
```

Comment:
Hmm, if the glitches are also there w/o the
revert, then why disabling this?

Reply:
the glitches are there if you use CPatch for vgaemu (independent of whether it's called from e_vgaemu_fault or directly from e_emu_pagefault), so Cpatch use for vgaemu needs to be disabled unfortunately (since it's much faster)